src/neo/Wallets/Wallet.cs

```diff
@@ -397,10 +397,9 @@ public long CalculateNetworkFee(StoreView snapshot, Transaction tx)
                     // Check verify cost
                     using ApplicationEngine engine = ApplicationEngine.Create(TriggerType.Verification, tx, snapshot.Clone());
                     if (engine.LoadContract(contract, "verify", CallFlags.None, true) is null)
-                        throw new ArgumentException($"The smart contract {contract.ScriptHash} haven't got verify method");
-                    engine.LoadScript(Array.Empty<byte>(), CallFlags.None);
```

Comment:
Should this line be removed?

Reply:
It's load an empty script (0Gas) i think that it's not needed